tests/framework/binding.h

```diff
@@ -1242,6 +1244,54 @@ inline VkBufferCreateInfo Buffer::CreateInfo(VkDeviceSize size, VkFlags usage, c
     return info;
 }
 
+class Tensor : public internal::NonDispHandle<VkTensorARM> {
+  public:
+    explicit Tensor();
+    explicit Tensor(const Device &dev, const bool is_copy_tensor = false);
+    explicit Tensor(const Device &dev, const VkTensorDescriptionARM &desc);
+    explicit Tensor(const Device &dev, const VkTensorCreateInfoARM &info);
+    ~Tensor() noexcept;
+    void destroy() noexcept;
+
+    // vkCreateTensor()
+    void init_no_mem(const Device &dev);
```

Comment:
```suggestion
    void InitNoMem(const Device &dev);
    
    BindToMem();
```

all of these need to follow function naming convention of the other bindings